app/model/knock/auth_token.rb

```diff
@@ -37,14 +38,24 @@ def options
 
     def claims
       {
-        exp: Knock.token_lifetime.from_now.to_i,
+        exp: token_lifetime,
         aud: token_audience
       }
```

Comment:
What do you think of modifying this method to something like the following?

```ruby
def claims
  _claims = {}
  _claims[:exp] = token_lifetime if verify_lifetime?
  _claims[:aud] = token_audience if verify_audience?
end
```

This way we don't include unnecessary `nil` values in the payload.